packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.types.ts

```diff
@@ -61,6 +61,12 @@ export interface IDatePickerProps extends IBaseProps<IDatePicker> {
    */
   ariaLabel?: string;
 
+  /**
+   * Whether or not the Textfield of the DatePicker is underlined.
+   * @default false
+   */
+  underlined?: boolean;
```

Comment:
my only thought here is if there is value in simply exposing the Textfield props. It does hide this mode a little bit, but avoids us deprecating it in the future if we decide to expose all the Textfield props.

Reply:
Hmm interesting thought. I'm not sure if we'd ever want to expose TextField props like mask, maskChar, prefix, suffix, errorMessage, etc., but maybe I'm just overlooking a use case for those. @betrue-final-final what do you think?

Reply:
nah, that's a good point. there are a TON of things in there that don't make sense to expose here. If anything, we'd want to share some lower level abstraction between datepicker and Textfield.

I'm good with this as is, and if we ever come to a situation where we need to change it, we'll do it then.